docker/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.7-alpine
 
 ARG DRIVER_PORT
 ENV DRIVER_PORT ${DRIVER_PORT:-8292}
+ARG SSL_DRIVER_PORT
+ENV SSL_DRIVER_PORT ${SSL_DRIVER_PORT:-8392}
```

Comment:
Do we need an alternative port for SSL? Can we just use the one port?

Reply:
yep